### PR TITLE
Ensure that Cordova is installed for `meteor [add | remove] cordova:*`.

### PR DESCRIPTION
**Pull request by abernix**
In the same spirit as the changes made in https://github.com/meteor/meteor/pull/8976.

Fixes: https://github.com/meteor/meteor/issues/9257.

### CODE REVIEW

File: tools/cli/commands-packages.js [hunk 1]
```diff
@@ -23,7 +23,10 @@ var packageMapModule = require('../packaging/package-map.js');
 var packageClient = require('../packaging/package-client.js');
 var tropohouse = require('../packaging/tropohouse.js');
 
-import * as cordova from '../cordova';
+import {
+  newPluginId,
+  splitPluginsAndPackages,
```
Comment (benjamn) [inline tools/cli/commands-packages.js]: Any reason not to import `ensureCordovaDependencies` here?

File: tools/cli/commands-packages.js [hunk 2]
```diff
@@ -1998,34 +2001,40 @@ main.registerCommand({
...
-      } else {
-        plugins[id] = version;
-        Console.info(`Added Cordova plugin ${id}@${version}.`);
-        changed = true;
+    function cordovaPluginAdd() {
+      let plugins = projectContext.cordovaPluginsFile.getPluginVersions();
+      let changed = false;
+
+      for (target of pluginsToAdd) {
+        let { id, version } =
```
Comment (benjamn) [inline tools/cli/commands-packages.js]: Some of these `let` declarations could be `const` declarations.

File: tools/cli/commands-cordova.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ import {
   CORDOVA_PLATFORMS,
   ensureDevBundleDependencies,
   filterPlatforms,
-} from '../cordova';
+} from '../cordova/index.js';
```
Comment (benjamn) [inline tools/cli/commands-cordova.js]: I support this explicitness. 👍

**Comment by tripflex** [comment]
Hey guys, i'm getting this error now after upgrading to 1.6.0.1 from 1.6.0 when attempting to add a cordova plugin from github:

```shell
~/c/d/m/f/src ❯❯❯ meteor add cordova:com.scott.serviceDiscovery@https://github.com/scottdermott/cordova-plugin-discovery.git#5431328fbb477d56d14f751fb39b679e7c631220                                                                                                                    ⏎ dev ✱
/Users/myles/.meteor/packages/meteor-tool/.1.6.0_1.1u0toj2.l6xqf++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:218
      throw error;
      ^

Error: Cannot find module 'cordova-registry-mapper'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Module.newPluginId (/tools/cordova/index.js:1:36)
    at Command.func (/tools/cli/commands-packages.js:2027:29)
    at /tools/cli/main.js:1522:15
```

**Pull request merged**